src/reducers/PagesReducer.js

```diff
@@ -21,7 +21,7 @@ export const PagesReducer = (state = {}, action) => {
       const { pageObject } = params;
 
       const pageInitialiser = getPageInitialiser(routeName);
-      const pageInitialState = pageInitialiser(pageObject);
+      const pageInitialState = pageInitialiser?.(pageObject);
```

Comment:
Is this valid?

Reply:
yes: https://github.com/tc39/proposal-optional-chaining

others too i.e. https://github.com/tc39/proposal-nullish-coalescing

pr added: https://github.com/openmsupply/mobile/pull/1393

Reply:
Nice, I like it!

Reply:
https://ponyfoo.com/articles/null-propagation-operator